src/integrations/feed/clients.ts

```diff
@@ -0,0 +1,167 @@
+import { FeedConfig, FeedResponse, IFeedClient } from './types';
+import fetch, { RequestInit } from 'node-fetch';
+import { fetchOptions as globalFetchOptions } from '../../http';
+import pRetry, { AbortError } from 'p-retry';
+import { IORedisPool } from '@dailydotdev/ts-ioredis-pool';
+import { Context } from '../../Context';
+
+const ONE_DAY_SECONDS = 24 * 60 * 60;
+
+/**
+ * Naive implementation of a feed client that fetches the feed from the feed service
+ */
+export class FeedClient implements IFeedClient {
```

Comment:
In which case would you use FeedClient directly and not the Cached version?
Just thinking we might want to restrict which one you can call?

Reply:
There can be temporary feeds which you will not want to cache and anyway caching is gonna move to feed service side. So I think it's better to keep it flexible and have both options

Reply:
Got it!
Just means keeping a good eye on review including anything feeds to see which ones are used and should be used.